doc/source/whatsnew/v1.5.0.rst

```diff
@@ -511,6 +511,12 @@ Strings
 - Bug in :meth:`str.startswith` and :meth:`str.endswith` when using other series as parameter _pat_. Now raises ``TypeError`` (:issue:`3485`)
 -
 
+Merge
```

Comment:
no need to add a new section. These headers are consistent between releases. please move to reshaping section.